massgov/mayflower
# Feature/dp 13100  topic card links

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

- Styling the topic cards to match the Invision mockups provided in DP-13100. 

## Related Issue / Ticket

- Jira Ticket DP-13100

## Steps to Test


*Go here*
http://localhost:3000/?p=pages-topic

*Do This*
- Scroll to the topic cards.

*You should See*
- The topic cards matching the mockups outlined in this Invision link: https://eotss.invisionapp.com/share/XQR3BDXJ869#/screens

1. 

## Screenshots
N/A


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* Section links and topic cards
